babel.config.js

```diff
@@ -1,34 +1,37 @@
 "use strict";
 
 module.exports = function (api) {
-  const env = api.env();
-  api.cache.using(() => env);
-
-  const plugins = ["@babel/plugin-proposal-class-properties"];
-
-  if (env === "production") {
-    plugins.push("emotion");
-  } else if (env === "development" || !env) {
-    plugins.push(["emotion", { sourceMap: true }]);
-  }
-
+  api.cache(true);
   return {
+    targets: {
+      // This is browserslist defaults
+      browsers: ["> 0.5%", "last 2 versions", "Firefox ESR", "not dead"],
+    },
     presets: [
       [
         "@babel/preset-env",
         {
           useBuiltIns: "entry",
-          corejs: 3,
+          corejs: require("core-js/package.json").version,
```

Comment:
We could use `babel.config.json` if the `corejs` version can be obtained via a magic string, e.g. `"corejs": "infer-from-node_modules"`